docs/faq.md

```diff
@@ -19,6 +31,21 @@
     - [Setting up PMM Server](setting-up/server/index.md)
     - [Setting up PMM Client](setting-up/client/index.md)
```

Comment:
does this go right to the "requirements" or just the page and the user has to figure it out (can't easily follow from git)...Can we recap the basic requirements here and use these links for more details?